google_chat_ros/scripts/google-chat.l

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env roseus
+
+(ros::load-ros-manifest "google_chat_ros")
+(ros::roseus "google_chat_eus_client")
+
+(defun send-google-chat-message (space message-type content &key (topic-name "google_chat_ros/rest") (wait nil))
```

Comment:
This comment is a matter of taste.

`message-type` argument in `(send-google-chat-message)` can be optional or key argument.
Like
```
(defun send-google-chat-message (space content &key (message-type "text") (topic-name "google_chat_ros/rest") (wait nil))
```

This is because I think that
- we use message-type "text" in many cases.
- command like the following is a bit confusing. (We need to remember the order of 3 strings.)
```lisp
(send-google-chat-message "YOUR_SPACE" "text" "Hello world from eus!")
```

Reply:
I agree with it, fixed